backend/src/database/favorite/favorite.service.ts

```diff
@@ -41,8 +29,8 @@ export class FavoriteService {
     return this.prisma.favorite.delete({
       where: {
         contentID_favoriteUserID: {
-          contentID,
-          favoriteUserID,
+          contentID: contentID,
+          favoriteUserID: favoriteUserID,
```

Comment:
This is not required because the object property shorthand of TypeScript allows you to pass a variable directly to an object if their keys are the same.

Reply:
I agree

Reply:
Ich habe das jetzt aber überall gemacht und will das jetzt nicht überall ändern. Außerdem finde ich das übersichtlicher, weil man dann direkt sieht was da passiert. Besonders wenn die beiden Namen unterschiedlich sind ist es ja zwingend notwendig.